src/components/policy/src/policy/src/policy_manager_impl.cc

```diff
@@ -1097,6 +1097,8 @@ void PolicyManagerImpl::OnChangedPrimaryDevice(
   if (!access_remote_->IsPrimaryDevice(device_id)) {
     std::string default_hmi;
     if (GetDefaultHmi(application_id, &default_hmi)) {
+      Subject who = { device_id, application_id };
```

Comment:
Why is this code appear after check for default HMI? It seems independent on default HMI level?